src/tests/build.proj

```diff
@@ -467,6 +472,72 @@
     <Message Importance="High" Text="App: $(FinalPath)" />
   </Target>
 
+  <!-- This target sets custom ILC properties to ensure correct values for the build -->
+  <Target Name="_SetCustomOSSpecificProps">
+    <PropertyGroup>
+      <_targetOS>$(TargetOS)</_targetOS>
+      <UseNativeAOTRuntime>true</UseNativeAOTRuntime>
+      <NativeLib>static</NativeLib>
+      <CustomNativeMain>true</CustomNativeMain>
```

Comment:
Why are these settings duplicated in `GroupBuildCmd` below? (Also why it's in a target?)

Reply:
Correct, all properties can be set in `Directory.Build.props`. Initially, it wasn't clear whether all MSBuild tasks shared the same .props file.